erpnext/buying/doctype/purchase_common/purchase_common.js

```diff
@@ -148,7 +148,12 @@ erpnext.buying.BuyingController = erpnext.TransactionController.extend({
 			frappe.model.round_floats_in(item, ["qty", "received_qty"]);
 			item.rejected_qty = flt(item.received_qty - item.qty, precision("rejected_qty", item));
 		}
-
+		
+		if(doc.is_return){
+			item.received_qty = item.qty;
+			item.rejected_qty = 0;
```

Comment:
You can not make it as Zero, user can also return the quantity from the rejected warehouse